packages/dev/src/commands/init.js

```diff
@@ -0,0 +1,50 @@
+import { spawn } from 'node:child_process';
+import { writeFile } from 'node:fs/promises';
+import { join } from 'node:path';
+
+import { CONFIG_FILE } from '../utils/getConfiguration.js';
+import { getCLIRoot } from '../utils/getMonorepoRoot.js';
+
+/**
+ * Performs one-time machine-level configuration tasks such as creating a blank config file.
+ */
+export async function init() {
+  const cliRoot = getCLIRoot();
+
+  /** @type import('../utils/getConfiguration.js').Configuration */
+  const configuration = {
+    $schema: join(cliRoot, 'packages', 'dev', 'schema.json'),
+    root: null,
+    activeInstance: 'default',
+    instances: {
+      default: {
+        secretKey: 'sk_REPLACE_WITH_SECRET_KEY',
+        publishableKey: 'pk_REPLACE_WITH_PUBLISHABLE_KEY',
+        fapiUrl: 'REPLACE_WITH_FAPI_URL',
+        bapiUrl: 'https://api.clerk.com',
```

Comment:
I put these in since these values are displayed on the configuration screen in the dashboard. I'm not totally sure of when a user would need to have these values available.

Reply:
Then let's remove it for now 👍